### PR TITLE
2.7 - Short plugin alias in shells

### PR DESCRIPTION
**Pull request by robertpustulka**
Here is an attempt to implement a feature mentioned in #5679 

Simple check for shell class existence in a plugin with the same name.

### CODE REVIEW

**Comment by AD7six** [comment]
Assuming that works/passes tests :+1:

File: lib/Cake/Test/test_app/Plugin/TestPlugin/Console/Command/TestPluginShell.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Short description for file.
```
Comment (markstory) [inline lib/Cake/Test/test_app/Plugin/TestPlugin/Console/Command/TestPluginShell.php]: Should remove this.

File: lib/Cake/Test/test_app/Plugin/TestPlugin/Console/Command/TestPluginShell.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://book.cakephp.org/2.0/en/development/testing.html CakePHP(tm) Tests
+ * @package       Cake.Test.TestApp.Plugin.TestPlugin.Console.Command
+ * @since         CakePHP(tm) v 1.2.0.7871
```
Comment (markstory) [inline lib/Cake/Test/test_app/Plugin/TestPlugin/Console/Command/TestPluginShell.php]: Should be 2.7.0.

**Comment by markstory** [comment]
PHPCS error has been fixed on master already.

**Comment by robertpustulka** [comment]
Is there anything broken in 2.7 branch? I rebased my branch to follow 2.7 and now travis fails.

**Comment by dereuromark** [comment]
I don't think this error is related to your changes. All looks still fine.

**Comment by markstory** [comment]
I can take a look at the build failure tonight, but it looks un-related to your changes.

**Comment by dereuromark** [comment]
Loos like it was introduced during the merge of master: https://travis-ci.org/cakephp/cakephp/builds/47322889

**Comment by markstory** [comment]
Strict error in 2.7 has been fixed.

**Pull request merged**